<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 	#333333</string>
  </property>
  <widget class="QPushButton" name="suc_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border-radius: 30;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <activeon>../icons/message/sucsesfully.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yandex Sans Display Thin</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>sucsesfully</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
